### PR TITLE
adding create geometries sample

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/CreateGeometries/CreateGeometries.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  // Add a graphic to the graphic collection - polyline with a simple line symbol.
+  m_graphicsOverlay->graphics()->append(new Graphic(createPolyline(), simpleLineSymbol, this));
+
+  // Add a graphic to the graphic collection - map point with a simple marker symbol.
+  m_graphicsOverlay->graphics()->append(new Graphic(createPoint(), simpleMarkerSymbol2, this));
+
+  // Add a graphic to the graphic collection - multi-point with a simple marker symbol.
+  m_graphicsOverlay->graphics()->append(new Graphic(createMultipoint(), simpleMarkerSymbol, this));
+
+  // Zoom to the extent of an envelope with some padding (10 pixels).
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/CreateGeometries/CreateGeometries.cpp]: You could say `10 device-independent pixels` here, or `DIPs`.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        // Add a graphic to the graphic collection - polyline with a simple line symbol.
+        graphicsOverlay.graphics.append(createGraphic(createPolyline(), simpleLineSymbol));
+
+        // Add a graphic to the graphic collection - map point with a simple marker symbol.
+        graphicsOverlay.graphics.append(createGraphic(createMapPoint(), simpleMarkerSymbol2));
+
+        // Add a graphic to the graphic collection - multi-point with a simple marker symbol.
+        graphicsOverlay.graphics.append(createGraphic(createMultipoint(), simpleMarkerSymbol));
+
+        // Zoom to the extent of an envelope with some padding (10 pixels).
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: Same here, `device-independent pixels`.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    function createEnvelope() {
+        // Return an envelope that covers the extent of the western United States.
+        return ArcGISRuntimeEnvironment.createObject("Envelope", {
+                                                         xMin: -123.0,
+                                                         yMin: 33.5,
+                                                         xMax: -101.0,
+                                                         yMax: 48.0,
+                                                         spatialReference: ArcGISRuntimeEnvironment.createObject("SpatialReference", {wkid: 4326})
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: You could use `SpatialReference.createWgs84()` as well.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                                                     });
+    }
+
+
+    function createPolygon() {
+        // Create a polygon builder
+        var polygonBuilder = ArcGISRuntimeEnvironment.createObject("PolygonBuilder", {
+                                                                       spatialReference: ArcGISRuntimeEnvironment.createObject("SpatialReference", {wkid: 4326})
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: You could use `SpatialReference.createWgs84()` as well.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+        polygonBuilder.addPointXY(-109.048, 36.998);
+
+        // Return the geometry.
+        return polygonBuilder.geometry;
+    }
+
+    function createPolyline() {
+        // Create a polyline builder
+        var polylineBuilder = ArcGISRuntimeEnvironment.createObject("PolylineBuilder", {
+                                                                        spatialReference: ArcGISRuntimeEnvironment.createObject("SpatialReference", {wkid: 4326})
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: You could use `SpatialReference.createWgs84()` as well.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+        // Return the geometry.
+        return polylineBuilder.geometry;
+    }
+
+    function createMapPoint() {
+        // Return a map point where the Esri headquarters is located.
+        return ArcGISRuntimeEnvironment.createObject("Point", {
+                                                         x: -117.195800,
+                                                         y: 34.056295,
+                                                         spatialReference: ArcGISRuntimeEnvironment.createObject("SpatialReference", {wkid: 4326})
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: You could use `SpatialReference.createWgs84()` as well.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+                                                     });
+    }
+
+    function createMultipoint() {
+        // Create a polygon builder
+        var multipointBuilder = ArcGISRuntimeEnvironment.createObject("MultipointBuilder", {
+                                                                          spatialReference: ArcGISRuntimeEnvironment.createObject("SpatialReference", {wkid: 4326})
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/CreateGeometries/CreateGeometries.qml]: You could use `SpatialReference.createWgs84()` as well.

**Comment by ldanzinger** [comment]
@michael-tims fixes pushed (screenshot, DIPs comment, wgs 84 helper)

**Pull request merged**